samples/zephyr/subsys/mgmt/mcumgr/smp_svr/prj.conf

```diff
@@ -39,3 +39,8 @@ CONFIG_MCUBOOT_UTIL_LOG_LEVEL_WRN=y
 
 # Disable debug logging
 CONFIG_LOG_MAX_LEVEL=3
+
+CONFIG_MCUBOOT_IMGTOOL_UUID_VID=y
```

Comment:
make this a board Kconfig fragment file, same with other 2, with a sysbuild Kconfig then it would be easier as VID one be in the sysbuild file and other 2 files would only have CID, there would not be an MCUboot file